public/homepage/stylesheets/style.less

```diff
@@ -1,21 +1,12 @@
-@import "variables.less";
-
-@font-face {
-  font-family: "Fira Medium";
-  src: url("https://code.cdn.mozilla.net/fonts/woff/FiraSans-Regular.woff") format("woff");
-  font-weight: normal;
-  font-style: normal;
+body {
+  font-family: 'Open Sans', sans-serif;
```

Comment:
this feels like a pretty shallow stack, do we not want a few more websafe sans fonts in there? Just falling straight to "whatever is sans-serif" when Open Sans fails is a bit drastic.